tests/unit/test_column_noise.py

```diff
@@ -315,11 +315,13 @@ def test_leave_blank(dataset: Dataset, fuzzy_checker: FuzzyChecker) -> None:
 
     data = dataset.data[[column_name]]
     NOISE_TYPES.leave_blank(dataset, config, column_name)
-    noised_data = dataset.data[column_name]
+    noised_data: pd.Series = dataset.data[column_name]
     # Calculate newly missing data, ie data that didn't come in as already missing
     data = data.squeeze()
-    noised_data = noised_data.squeeze()
-    orig_non_missing_idx = data.index[(data.notna()) & (data != "")]
+    # mypy can't know that squeeze() will not produce a scalar value
+    noised_data = noised_data.squeeze()  # type: ignore [assignment]
```

Comment:
Why are we squeezing a Series anyway? That seems wrong. If we end up with a scalar the code below will fail (e.g. `noised_data.index[...]` in line 325

Reply:
I've removed the squeeze (tests still pass)